cohttp-lwt/src/dune

```diff
@@ -4,4 +4,4 @@
  (synopsis "Lwt backend")
  (preprocess
   (pps ppx_sexp_conv))
- (libraries lwt uri cohttp logs logs.lwt))
+ (libraries lwt lwt.unix uri cohttp logs logs.lwt))
```

Comment:
It's probably **not** OK to add Lwt.unix here. 
 The readme file says : 

> Cohttp_lwt exposes an OS-independent Lwt interface, which is used by the [Mirage](https://mirage.io/) interface to generate standalone microkernels (use the cohttp-mirage subpackage).

Reply:
Indeed. What about using something with `Lwt.task` instead of the `Lwt_unix.sleep`?

Reply:
Changed [here](https://github.com/gabrielmoise17/ocaml-cohttp/pull/1/commits/c81c7f3d17fff1aa26a823c07b1e1b57e2377ea2)